src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmissingwhitespaceafterasterisk/InputJavadocMissingWhitespaceAfterAsteriskInvalid.java

```diff
@@ -0,0 +1,62 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmissingwhitespaceafterasterisk;
+
+
+
+/**
+ *Some Javadoc.
```

Comment:
`Some Javadoc` ==> `violation`.

please do the same for all, to clearly define line that will have violation.
Please apply this to all other cases.

Reply:
done